typescript/utils/package.json

```diff
@@ -17,6 +17,7 @@
     "chai": "4.5.0",
     "mocha": "^10.2.0",
     "prettier": "^2.8.8",
+    "sinon": "^19.0.2",
```

Comment:
Please use the same version as in other packages (so 13.0.2 like in the SDK) or upgrade it for all packages across the whole monorepo. See the SDK's package.json for an example. You'll also want to add these:
```
    "@types/sinon": "^17.0.1",
    "@types/sinon-chai": "^3.2.12",
    ```

Reply:
And then a separate issue: you need to run `yarn` after changing a package.json so the yarn.lock file can be updated. That's why CI is failing

Reply:
I don't know why, but in my MacOS, it cannot find patch for `typechain`
```
Error: typechain@patch:typechain@npm%3A8.3.2#~/.yarn/patches/typechain-npm-8.3.2-b02e27439e.patch: 
 ENOENT: no such file or directory, open '~/.yarn/patches/typechain-npm-8.3.2-b02e27439e.patch'
```

Reply:
After use relative path for patch, I've updated `yarn.lock` successfully~

Reply:
@tiendn I think you're using an outdated version of yarn. That's also why the yarn.lock file has a huge amount of changes. It should have minimal changes. Please ensure you're using yarn 4.0.2 as is specified in the .yarnrc.yml file. If you're using any version of yarn 4, it _should_ just set it to 4.0.2 automatically when run in the monorepo

Reply:
tks, I've fixed it